### PR TITLE
BUG Fix travis / unit tests

### PR DESCRIPTION
**Pull request by tractorcow**

### CODE REVIEW

**Comment by tractorcow** [comment]
Eh, travis isn't setup properly on this repo. I need someone with admin privileges. =/

**Comment by tractorcow** [comment]
Someone check if https://travis-ci.org/profile/silverstripe has travis enabled. Something is up.

**Comment by sminnee** [comment]
I've turned on travis for this project but can't seem give it a nudge to rebuild.

**Comment by sminnee** [comment]
Is there a .travis.yml file in there? I don't see it...

**Comment by sminnee** [comment]
Oops I see that's already merged. Travis should be all set up now; maybe we merge this in to trigger it?

**Comment by tractorcow** [comment]
Actually there IS ment to be a .travis.yml. I somehow neglected to include it in the PR somehow. :)

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ env:
 matrix:
   include:
     - php: 5.4
-      env: DB=MYSQL CORE_RELEASE=master
+      env: DB=MYSQL CORE_RELEASE=3
```
Comment (tractorcow) [inline .travis.yml]: Because master is now 4.0 we test against 3 instead.

**Comment by tractorcow** [comment]
Travis looks good finally. :)

**Comment by tractorcow** [comment]
slackbot made me do it

**Pull request merged**